engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.controller.js

```diff
@@ -95,6 +95,14 @@ angular.module('Bastion.products').controller('ProductsController',
             ProductBulkAction.syncProducts(getBulkParams(), success, bulkError);
         };
 
+        $scope.goToDiscoveries = function (){
+            nutupane.table.rows = [];
+            nutupane.table.resource.results = [];
+            nutupane.table.resource.total = 0;
+            nutupane.table.resource.subtotal = 0;
```

Comment:
Does simply calling `nutupane.invalidate()` not work here?

Reply:
No, that calls the TableCache.removeTable() which blows away the whole table. I could probably remove the subtotal (line 102) but that wouldn't really help clean it up.

Reply:
Oh I see.